hiku/expr/core.py

```diff
@@ -142,17 +142,7 @@ def expr(*args: P.args, **kw: P.kwargs) -> _Func:
 
         expr.__def_name__ = name  # type: ignore[attr-defined]
         expr.__def_body__ = fn  # type: ignore[attr-defined]
-
-        if len(types) == 1 and isinstance(types[0], str):
```

Comment:
This if was to support `edn` in defines - `@define("[[:foo :bar]]")`. 

Now we only supporting actual types such as `@define(String)` or `@define(Record[{"foo": String, "bar": Integer}])`